vuln_due_date_cleanup/due_date_cleanup.py

```diff
@@ -0,0 +1,271 @@
+        print(f"{sys.argv[0]} <risk_token/API_key> <risk_meter_id> <number_of_days>")
+        print("==========================================================================================================")
+        sys.exit(1)
+elif len(sys.argv[1]) < 64:
+    print("==========================================================================================================")
+    print("Provided API Key (Risk_Token) is missing characters. Provided one has", len(sys.argv[1]), "as a normal one has 64. Missing", 64 - len(sys.argv[1]),"characters.")
+    print("==========================================================================================================")
+    sys.exit(1)
+elif int(sys.argv[3]) > 7:
+    print("==========================================================================================================")
+    print("Number of days to look back can't be more than 7 days. You chose ", sys.argv[3])
+    print("==========================================================================================================")
+    sys.exit(1)
+
+RiskToken = sys.argv[1]
+RiskMeter = sys.argv[2]
+
+# Created to validate if the vuln score changed during the last 24 hours or more (you can custom it)
+num_days = int(sys.argv[3])
+start_time = datetime.today()
+yesterday_datetime = (datetime.today() - timedelta(num_days)).strftime('%Y-%m-%d %H:%M:%S+00:00')
+
+# Used to bulk update the CVE's due dates
+base_uri3 = 'vulnerabilities/bulk'
+url3 = ''.join([base_url, base_uri3])
+
+# Map of the CVE_IDs for a specific CVE_Name
+cve_id = {}
+
+# Parameters created to validate if there are any CVEs to change
+cve_id_exist = ''
+cve_id_exist3 = []
+cves_numbers_v1 = 0
+cves_numbers_v3 = 1
+cve_check = 0
+
+# Function to bulk update the CVE IDs and add the needed comments
+def bulk_func(d_id,d_cve_id, d_url, d_headers, d_num_days, d_list2, d_start_time):
+    print("....")
+    print("==========================================================================================================")
+    print("Removing the due_date from the needed vulnerabilities.")
+    
+    if d_id == 1:
+        data2 = '{ "vulnerability_ids" : ' + str(d_cve_id) + ', "vulnerability" : { "due_date": "" } }'
+    elif d_id == 2:
+        data2 = '{ "vulnerability_ids" : [' + str(d_cve_id) + '], "vulnerability" : { "due_date": "" } }'
+
+    return_data = requests.put(d_url, headers=d_headers, data=data2)
+    print("....")
+    print("URL3:     " + d_url)
+    print("Data:     " + data2)
+    print("Response: " + str(return_data))
+    print("....")
+
+# Setup Data Dump
+headers = {
+    'X-Risk-Token':  RiskToken,
+    'Content-type': 'application/json',
+}
+
+data = '{ "status" : ["open"], "search_id" : ' + str(RiskMeter) + ', "export_settings" : { "format": "json", "model": "vulnerability" } }'
+base_uri = '/data_exports'
+url = ''.join([base_url, base_uri])
```

Comment:
You might consider changing to: 
`export_uri = '/data_exports'`
`url = base_url + export_uri'`
Also, the slash is not needed in the URI since it is in base_url.

Reply:
Working on it.